phaseshifts/lib/phsh.py

```
@@ -478,9 +462,7 @@ def s16(r, rt, v, z, mtz=sys.stdin, zph=sys.stdout):
     # read format used for input of r vs. v[r) or r vs. r*v[r)
     # (v is assumed positive)
     for i in range(1, 200 + 1):  # 16
-        rs[i], zs[i] = [
-            t(s) for t, s in zip((float, float), mtz.readline().split()[:1])
-        ]
+        rs[i], zs[i] = [t(s) for t, s in zip((float, float), mtz.readline().split()[:1])]
```

Ruff (0.14.10)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)